packages/shared/src/components/squads/SquadMemberShortList.tsx

```diff
@@ -25,7 +25,7 @@ function SquadMemberShortList({
       type: LazyModal.SquadMember,
       props: {
         squad,
-        placeholderAmount: squad?.membersCount,
+        placeholderAmount: memberCount ? memberCount : squad?.membersCount,
```

Comment:
Why do we use two implementations if squad already holds membersCount?
Maybe we can merge it from the top level rather.